easybuild/easyconfigs/b/Bowtie2/Bowtie2-2.2.9-intel-2016b.eb

```diff
@@ -0,0 +1,49 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# Modified by: Robert Schmidt <rjeschmi@gmail.com>
+# Ottawa Hospital Research Institute - Bioinformatics Team
+# Modified by: Adam Huffman <adam.huffman@crick.ac.uk>
+# The Francis Crick Institute
+# Modified by: Kurt Lust, UAntwerp: Patch for better optimization with the
+# Intel compilers, based on 
+# https://www.allinea.com/blog/201501/tuning-bowtie2-better-performance
```

Comment:
no longer patched, and I'm not sure if advertising Allinea is required for something as simply as making sure the compilation flags provided by EasyBuild are being used?